indexd/index/blueprint.py

```diff
@@ -295,6 +295,30 @@ def post_index_blank_record():
     return flask.jsonify(ret), 201
 
 
+@blueprint.route("/index/blank/<path:record>", methods=["POST"])
+def add_index_blank_record_version(record):
+    """
+    Create a new blank version of the record with this GUID.
+    Authn/authz fields carry over from the previous version of the record.
+    Only uploader and optionally file_name fields are filled.
+    Returns the GUID of the new blank version and the baseid common to all versions
+    of the record.
+    """
+    uploader = flask.request.get_json().get("uploader")
+    file_name = flask.request.get_json().get("file_name")
+    if not uploader:
+        raise UserError("no uploader specified")
+
+    # authorize done in add_blank_version for both the old and new authz
```

Comment:
i see this comment was copied from `add_index_record_version` and it actually looks like `add_version` is _not_ checking both old and new authz! that worries me

Reply:
I see -- how big of a deal is that??

Reply:
sorry 🤦‍♀️ i checked the code again and it does check both. woops

Reply:
haha no way! Ok phew ✨